examples/limit-requests/Readme.md

```diff
@@ -0,0 +1,32 @@
+### Example to limit requests
+
+This is a working example to limit the concurrency of requests.
+
+#### Motivation
+
+Using eth-fun without any concurrency can choke the RPC endpoint.
+
+#### Overview
+
+This example uses the [p-limit library](https://github.com/sindresorhus/p-limit) to limit concurrency.
+
+```js
+const pool = pLimit(concurrency);
+const promises = [];
+
+for (let i = 0; i < totalRequests; i++) {
+  promises.push(pool(() => blockNumber(options)));
+}
+```
```

Comment:
it may make sense that we explain how to resolve these promises with `Promise.allSettled`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Reply:
I have added a comment to explain it.